src/translator/parse/elements/dropdown/dropdown-schema-parser.ts

```diff
@@ -1,4 +1,5 @@
 import { ISelectSchema } from "@lifesg/web-frontend-engine/components/fields";
+import { isEmpty } from "lodash";
```

Comment:
use sub-directory import to reduce import size
```suggestion
import isEmpty from "lodash/isEmpty";
```